### PR TITLE
Update About Me content, image cycling on button click, and div arrangement.

### PR DESCRIPTION
**Pull request by ellisvivian**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -17,7 +17,8 @@
  */
 function addFunFact() {
   const fun_facts =
-      ['My favorite color is purple.', 'I have a little brother.', 'I am half Chinese.', 'I am 18 years old.'];
+      ['My favorite color is purple.', 'I have a little brother.', 'I am half-Chinese.', 'I am 18 years old.', 'My elbows are double-jointed.',
```
Comment (usaeed-w) [inline portfolio/src/main/webapp/script.js:20]: Recommend putting each element of the array on a new line. 

General guidance: If the elements all fit on one line, then one line is fine. However, if they wrap over multiple lines, then each element on its own line is preferred.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +27,93 @@ function addFunFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fun_fact;
 }
+
+let count;
+let timer;
+
+/*
+ * Displays the intital image and description once the page is loaded.
```
Comment (usaeed-w) [inline portfolio/src/main/webapp/script.js:35]: Nit: intital vs initial

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +27,93 @@ function addFunFact() {
...
+    count = -1;
+    start();
+}
+
+/*
+ * Changes the image and description displayed on the page.
+ */
+function autoSlideshow() {
+    let experiences = document.getElementsByClassName('experience');
+    console.log(experiences);
```
Comment (usaeed-w) [inline portfolio/src/main/webapp/script.js:47]: Once you are satisfied that the logic works, recommend getting rid of the log statements -- here and elsewhere in Javascript code.

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -26,3 +27,93 @@ function addFunFact() {
...
+ */ 
+// function changeDisplay() {
+//     if (count == 3) {
+//         count = 0;
+//     } else {
+//         count ++;
+//     }
+//     document.getElementById('img').src = image_sources[count];
+//     document.getElementById('description').innerText = descriptions[count];
+// }
```
Comment (usaeed-w) [inline portfolio/src/main/webapp/script.js:119]: Recommend getting rid of all the dead code.

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,41 +1,161 @@
...
+    padding: 5px 10px;
+    font-weight: 550;
+    border-radius: 4px;
+    transition: ease background-color 0.1s, ease color 0.1s;
 }
 
+.button:hover {
+    background-color: var(--box-color);
+    color: var(--accent-color);
+}
```
Comment (usaeed-w) [inline portfolio/src/main/webapp/style.css:161]: Add a newline at the end of the file.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (ellisvivian) [reply portfolio/src/main/webapp/script.js]: Done! (see commit "Address requested changes...")

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (ellisvivian) [reply portfolio/src/main/webapp/script.js]: Done! (see commit "Address requested changes...")

File: portfolio/src/main/webapp/script.js (hunk 3 above)
Comment (ellisvivian) [reply portfolio/src/main/webapp/script.js]: Done! (see commit "Address requested changes...")

File: portfolio/src/main/webapp/script.js (hunk 4 above)
Comment (ellisvivian) [reply portfolio/src/main/webapp/script.js]: Done! (see commit "Address requested changes...")

File: portfolio/src/main/webapp/style.css (hunk 5 above)
Comment (ellisvivian) [reply portfolio/src/main/webapp/style.css]: Done! (see commit "Address requested changes...")

**Pull request merged**